components/ContractStatus/Common.tsx

```diff
@@ -26,18 +26,48 @@ export const Fieldset: FunctionComponent = ({ children }) => {
 
 type SummaryProps = {
   disclosureState: DisclosureStateReturn;
+  contractRate: string;
+  direction: 'negative' | 'positive';
 };
 export function Summary({
-  children,
   disclosureState,
-}: PropsWithChildren<SummaryProps>) {
+  direction,
+  contractRate,
+}: SummaryProps) {
+  const adjustedContractRate = useMemo(
+    // Design could not squeeze into normal table, this hack
```

Comment:
Is this still relevant? I thought we changed the wording to get it to fit?

Reply:
This is as-designed